### ISSUE TITLE
CSV export wrong date

### ISSUE THREAD

**Issue by kurfuerst1982** [Swing-team/pendulums-web-client#32]
Input data in the web app and exported data are not the same.
Data in the web app are ok, but if you export the data (csv or json) the date isn´t correct.

Maybe problem with the internal Unix timestamp and my time zone, when I export the data.

b.r. gernot

**Comment by arminghm** [comment]
Did you change your timezone recently?

**Comment by kurfuerst1982** [comment]
no, have not, but I checked it now again.
And the time is now ok. Only the set date range is not the same , see pdf.
But this is not so the prob. Thanks for the fast reaction.
[bugrep_pendulums_web.pdf](https://github.com/Swing-team/pendulums-web-client/files/3734482/bugrep_pendulums_web.pdf)

b.r. gernot

**Comment by arminghm** [comment]
Oh, you say that the exported data is not in the range that you selected from diagram?

**Comment by kurfuerst1982** [comment]
Yes, but I think this is normal. But for month evaluation it will be good, only to export the selected date range.
